test/core/fixtures/transformation/es6.template-literals/expression-first/expected.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+var t = "" + foo + "bar";
+
+var foo = 5;
+var bar = 10;
+var baz = 15;
+
+var example = 1 + ("" + foo + bar + baz);
```

Comment:
This puts the nail in the coffin for this. There needs to be at least 1 explicit string in each binary expression.